dungeon.js

```diff
@@ -1367,14 +1379,17 @@ function hideConfig() {
 
 function saveConfig() {
     var valid = updateNumFloors();
+    valid &= updateSideLength();
     valid &= updateCellTypes();
     valid &= updateCellMarks();
     valid &= updateEdgeMarks();
 
     if (!valid) {
         return; // don't update or close so user can correct
     }
-
+    
+    redrawCanvas();
```

Comment:
nit - move to line about `redrawMap` instead

Reply:
Do you mean to relocate the function `redrawCanvas()` down to be near `redrawMap()`?

Reply:
yes pls, just so they're in logical groups